### PR TITLE
Note on variable names

### PR DESCRIPTION
**Pull request by cancan101**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -172,3 +172,7 @@ It is acceptable to use the following section divider when it aids clarity. For
 ```python
 # -----------------------------------------------------------------------------
 ```
+
+## Variable names
+
+Do not use one letter variable names, except for `i` or `j` in loops. Using `k` or `v` instead of `key` or `value`, for example in a `dict` comprehension is never appropriate.
```
Comment (itajaja) [inline README.md:178]: 👎 https://github.com/golang/go/wiki/CodeReviewComments#variable-names

`with open(file) as f` and similar is appropriate for me

Comment (cancan101) [reply README.md:178]: how about if we have a carve out for "readers" as well?

Comment (taion) [reply README.md:178]: Sure – I don't know how much we need to codify this, but something like `for k, v in foo.items()` looks quite bad to me, especially when the comprehension is not a one-liner.

Comment (taion) [reply README.md:178]: I suspect this would be better handled as "use common sense".

Comment (itajaja) [reply README.md:178]: quoting from the go doc:

> the further from its declaration that a name is used, the more descriptive the name must be

this makes total sense to me. so if you have

```py
return (f'{k}--{v}' for k, v in foo.items())
```

this is OK, but if you hava

```py
for k, v in foo.items():
   ... 20 lines block
```

then you probably want a more descriptive name

Comment (taion) [reply README.md:178]: right – i'm not sure we need to codify this here; it's just general best practices

Comment (itajaja) [reply README.md:178]: I agree there is a value in style guide being succinct

**Comment by cancan101** [comment]
Ok, do we want to close this?

**Pull request closed without merging**